### PR TITLE
GStreamer plugin

### PR DESCRIPTION
**Pull request by tianjunwork**
Signed-off-by: Jun Tian <jun.tian@intel.com>

### CODE REVIEW

**Comment by tianjunwork** [comment]
User guide and api.h doesn't match. Hard to figure out the APIs.

**Comment by kylophone** [comment]
We may want to add a Travis job for this, similar to the job that tests the ffmpeg patch.

**Comment by 1480c1** [comment]
If someone could tell me how, I can add it

**Comment by tianjunwork** [comment]
> Could you add the doxygen documentation?

Strictly speaking, the plugins are not part of AV1 project even in the same repo. It is a standalone project.
Hi, for long term the plugin will upstream to GStreamer. It should follow the documentation style of GStreamer.

**Comment by tianjunwork** [comment]
> If someone could tell me how, I can add it

Thank you:). Let me add this one. Could you help to review? Thx.

File: .travis.yml [hunk 1]
```diff
@@ -56,3 +58,17 @@ jobs:
...
+     script:
+     # Build and install SVT-AV1
+     - cd $TRAVIS_BUILD_DIR
+     - cd Build
+     - cmake ..
+     - make -j$(nproc)
+     - sudo make install
+     # Build GST-SVT-AV1 plugin
+     - cd $TRAVIS_BUILD_DIR/gstreamer-plugin
+     - cmake
```
Comment (1480c1) [inline .travis.yml]:
```suggestion
     - cmake .
```
Make sure to add a directory

**Pull request merged**